src/library_wasmfs.js

```diff
@@ -25,6 +25,29 @@ FS.createPreloadedFile = FS_createPreloadedFile;
 #endif
   ],
   $FS : {
+    ErrnoError: null,
+    throwError: (errno) => {
+      if (!FS.ErrnoError) {
+        FS.ErrnoError = /** @this{Object} */ function ErrnoError(code) {
```

Comment:
```suggestion
        FS.ErrnoError = function ErrnoError(code) {
```
This is what I mean by removing the closure annotation.